indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/PendingTaskBasedWorkerProvisioningStrategy.java

```diff
@@ -58,6 +60,34 @@
 
   private static final String SCHEME = "http";
 
+  @Nullable
+  static DefaultWorkerBehaviorConfig getWorkerBehaviorConfig(
```

Comment:
Minor: `getDefaultWorkerBehaviorConfig` would be clearer.

Reply:
Changed